apps/ejabberd/src/ejabberd.app.src

```diff
@@ -39,7 +39,8 @@
                   fast_tls,
                   usec,
                   uuid,
-                  xmerl
+                  xmerl,
+                  worker_pool
```

Comment:
There's actually only one module using Poolboy in MongooseIM (`mongoose_http_client`).